src/utils/errorHandler.js

```diff
@@ -64,26 +59,29 @@ function normaliseError(error, customMessage, analyticsInstance) {
     }
   }
 
-  const customeErrMessagePrefix = customMessage || '';
-  return `[handleError]::${customeErrMessagePrefix} "${errorMessage}"`;
+  const customErrMessagePrefix = customMessage || '';
+  return `[handleError]::${customErrMessagePrefix} "${errorMessage}"`;
 }
 
 function handleError(error, customMessage, analyticsInstance) {
   let errorMessage;
+
   try {
     errorMessage = normaliseError(error, customMessage, analyticsInstance);
   } catch (err) {
     logger.error('[handleError] Exception:: ', err);
     logger.error('[handleError] Original error:: ', JSON.stringify(error));
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify 
`JSON.stringify(error)` this can throw errors when there is a circular dependency. should we care about this?

Reply:
this is an open bug we have as the integrations seem to have an instance of rudderanalytics per integration instance causing this to blow up in firefox. non related to bugsnug